### DISCUSSION TITLE
Playbook Disable

### DISCUSSION THREAD

**Discussion by argwfm** [Security-Onion-Solutions/securityonion#9636] [Unsupported Versions] [answered]
Is there a `global.sls` flag or some other way to disable Playbook functionality?

**Comment by InfosecGoon** [comment]
You could disable anonymous accounts, which would keep analysts from being able to log into it.

https://docs.securityonion.net/en/2.3/playbook.html#disable-anonymous-access-and-create-user-accounts

**Comment by argwfm** [comment, marked as answer]
Was able to disable by modifying the manager pillar w/ `playbook: 0`.
Then updated so-status.conf and commented out `#so-playbook` and `#so-mysql`.

![image](https://user-images.githubusercontent.com/99749555/217850112-3e88a2f2-0f76-489b-a4ff-b529f4ee9c9d.png)
